opdracht1.js

```diff
@@ -0,0 +1,75 @@
+// #### Opdracht 1 - Array Methoden
+
+
+// * **Opdracht 1a:** Gebruik een array-methode om een array te maken met alle tv-type namen. Log de uitkomst in de
+// console.
+
+// STAPPENPLAN:
+// Met de map-methode door de inventory array itereren
+// Bij elke iteratie alleen de type-property van de betreffende entry aan de return meegeven en deze opslaan in een nieuwe array
+// De nieuwe array loggen
+
+const types = inventory.map((tv) => {
+    return tv.type;
+})
+
+console.log(types);
+
+
+// * **Opdracht 1b:** Gebruik een array-methode om alle tv's te verzamelen (de hele objecten) die volledig uitverkocht
+// zijn. Log de uitkomst in de console.
+
+// STAPPENPLAN:
+// Met de filter-methode door de inventory array itereren
+// Bij elke iteratie controleren of de entry aan de conditie (uitverkocht) voldoet. Zo ja, een true meegeven aan de return
+// De volledige entry wordt in een nieuwe array gezet, deze array dus loggen
+
+const soldOut = inventory.filter((tv) => {
+    return tv.originalStock - tv.sold === 0;
+})
+
+console.log(soldOut);
+
+
+// * **Opdracht 1c:** Gebruik een array-methode om alle tv's te verzamelen (de hele objecten) die over AmbiLight
+// beschikken. Log de uitkomst in de console.
+
+// STAPPENPLAN:
+// Met de filter-methode door de inventory array itereren
+// Bij elke iteratie controleren of de entry aan de conditie (TV met abilight) voldoet. Zo ja, een true meegeven aan de return
+// De volledige entry wordt in een nieuwe array gezet, deze array dus loggen
+
+const withAmbiLight = inventory.filter((tv) => {
+    return tv.options.ambiLight;
+})
+
+console.log(withAmbiLight);
+
+
+// * **Opdracht 1d:** Schrijf een functie die alle tv's van laagste naar hoogste prijs sorteert. Log de uitkomst in de
+// console.
+
+// STAPPENPLAN:
+// Met de sort-methode door de inventory array itereren
+// Bij elke iteratie de prijs van de huidige entry met de waarde van de vorige entry vergelijken
+// Als de prijs van de huidige entry lager is dan de prijs van de vorige entry een -1 meegeven aan de return. Andersom wordt er een 1 aan de return meegegeven. Als de prijs gelijk is wordt er een 0 aan de return meegegeven.
+// Bovenstaande omschrijven naar een functie met 1 parameter
+// De functie aanroepen en de inventory-array als argument meegeven
+// De bestaande array wordt aangepast/gesorteerd, deze array dus loggen
+
+function sortPriceLowestToHighest(listWithInventory) {
+    listWithInventory.sort((a, b) => {
+        if (a.price < b.price) {
+            return -1;
+        } else if (a.price > b.price) {
+            return 1;
+        } else {
+            return 0;
+        }
+    })
```

Comment:
Deze had nog iets versimpeld kunnen worden
```suggestion
    listWithInventory.sort((a, b) => a.price - b.price);
```